### PR TITLE
Feat allow rspec rails

### LINKED ISSUE #49: NoMethodError: undefined method `version' for nil:NilClass
### Tell us about your environment

**Ruby Version:**
> ruby 2.2.6p396

**Framework Version (RSpec, Minitest, FactoryGirl, Rails, whatever):**
> RSpec (via rspec-rails v 3.6.0)

**TestProf Version:**
> test-prof (0.4.4)

### What actually happened?
```
NoMethodError:
  undefined method `version' for nil:NilClass
```
This is due to the fact that it is looking for 'rspec' gem specifically instead of 'rspec' or 'rspec-rails'

### PR DESCRIPTION
**Pull request by desoleary**
Fixes #49 

Found when using `rspec-rails` gem that `Gem.loaded_specs['rspec']` returned nil
In this case, we should allow for  use of `rspec` or `rspec-rails`

### CODE REVIEW

**Comment by desoleary** [comment]
@palkan PTAL

File: lib/test_prof/recipes/rspec/let_it_be.rb [hunk 1]
```diff
@@ -16,7 +16,9 @@ def module_for(group)
 
       # Only works with RSpec 3.2.0
       def supported?
-        TestProf::Utils.verify_gem_version('rspec', at_least: '3.2.0')
+        required_min_version = '3.2.0'
+        TestProf::Utils.verify_gem_version('rspec', at_least: required_min_version) ||
```
Comment (palkan) [inline lib/test_prof/recipes/rspec/let_it_be.rb]: Let's just verify `rspec-core` instead, that should work for both Rails and non-Rails projects

Comment (palkan) [reply lib/test_prof/recipes/rspec/let_it_be.rb]: Hi, @desoleary! Have you seen my comment?

**Comment by palkan** [comment]
Closed by https://github.com/palkan/test-prof/commit/dcc63535cebfa896455006d0f8419f9801434e20

**Pull request closed without merging**